checker/votour.ml

```diff
@@ -14,22 +14,29 @@ open Values
 
 let max_string_length = 1024
 
+type command =
+| CmdParent
+| CmdChild of int
+| CmdSort
+
 let rec read_num max =
   let quit () =
     Printf.printf "\nGoodbye!\n%!";
     exit 0 in
   Printf.printf "# %!";
   let l = try read_line () with End_of_file -> quit () in
-  if l = "u" then None
-  else if l = "x" then quit ()
-  else
+  match l with
+  | "u" -> CmdParent
+  | "s" -> CmdSort
+  | "x" -> quit ()
+  | _ ->
     match int_of_string l with
     | v ->
       if v < 0 || v >= max then
         let () =
           Printf.printf "Out-of-range input! (only %d children)\n%!" max in
         read_num max
-      else Some v
+      else CmdChild v
     | exception Failure _ ->
       Printf.printf "Unrecognized input! <n> enters the <n>-th child, u goes up 1 level, x exits\n%!";
```

Comment:
Missing `s sorts`?
Printing the commands should probably be shared with the other print at line 415(master)

Reply:
There should be a "help" command rather, but it's adding cruft atop of this PR. I'll do it.

Reply:
Done.